### ISSUE TITLE
Incorrect documentation for pure species property fits

### ISSUE THREAD

**Issue by speth** [Cantera/cantera#1488]
**Problem description**

The mathematical descriptions of the fitting functions used for the pure species viscosities, thermal conductivities, and binary diffusion coefficients are incorrect and do not correspond to the implementation of these fits in the code.

Compare these descriptions:
- [fitProperties](https://cantera.org/documentation/dev/doxygen/html/d8/d58/classCantera_1_1GasTransport.html#a267d20cdea7486fe84e722ee82d84b20)
- [fitDiffCoeffs](https://cantera.org/documentation/dev/doxygen/html/d8/d58/classCantera_1_1GasTransport.html#a384e3fa8d986dff5c70492d5be1a1d62)

With these functions:

https://github.com/Cantera/cantera/blob/1f1751b16c6f73234984b88118a094f724f05c0c/src/transport/GasTransport.cpp#L108-L125

https://github.com/Cantera/cantera/blob/1f1751b16c6f73234984b88118a094f724f05c0c/src/transport/GasTransport.cpp#L126-L150

https://github.com/Cantera/cantera/blob/1f1751b16c6f73234984b88118a094f724f05c0c/src/transport/MixTransport.cpp#L32-L49

**Additional context**

This problem was originally reported on the Users' Group: https://groups.google.com/g/cantera-users/c/_tUZCjeQjUU

**Comment by ischoegl** [comment]
Fwiw, the relevant passage for conductivity is https://github.com/Cantera/cantera/blob/22d760cbfad437c8d7d0ae7bca038d218b1be8b2/src/transport/MixTransport.cpp#L114-L127

**Comment by jsagg931** [comment]
Please, also beware that the coefficients obtained with `get_binary_diff_coeffs_polynomial` are scaled with the pressure in Pascals. That is, one should do


```
P = 1.01325e5  # Pa
T = 2000  # K
T_array = [np.log(T)**n for n in range(0,4)]
i = 0
j = 0
D_ij = (np.dot(scheme.get_binary_diff_coeffs_polynomial(i,j),T_array)) * np.sqrt(T) * T / P 

````
 I am unsure if that is also the case of `GasTransport::updateDiff_T`, it varies on the definition of the fitting diffusion. I believe the documentation should be updated accordingly. Additionally, in these fits there isn't any dependency on pressure, is that right? I don't see it in `GasTransport::fitDiffCoeffs`.

Thanks!

**Comment by speth** [comment]
Yes, you're correct, the pressure dependence is not indicated correctly in the docstring for `fitDiffCoeffs`. The fitting coefficients are pressure independent, as are the values that are stored internally. However, the actual binary diffusion coefficients scale with $1/p$.

The docstring for `getBinaryDiffCoeffs` is ugly but technically correct, if you can parse `d[ld*j + i] = rp * m_bdiff(i,j);` as indicating that the stored value is multiplied by the reciprocal of the pressure.
